src/Custom/Audio/OpenAIAudioModelFactory.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenAI.Audio;
+
+/// <summary> Model factory for models. </summary>
+public static partial class OpenAIAudioModelFactory
+{
+    /// <summary> Initializes a new instance of <see cref="OpenAI.Audio.AudioTranscription"/>. </summary>
+    /// <param name="duration"> The duration of the input audio. </param>
+    /// <returns> A new <see cref="OpenAI.Audio.AudioTranscription"/> instance for mocking. </returns>
+    public static AudioTranscription AudioTranscription(string language = null, TimeSpan? duration = null, string text = null, IEnumerable<TranscribedWord> words = null, IEnumerable<TranscribedSegment> segments = null)
+    {
+        words ??= new List<TranscribedWord>();
+        segments ??= new List<TranscribedSegment>();
+
+        return new AudioTranscription(
+            task: default,
+            language,
+            duration,
+            text,
+            words.ToList(),
+            segments.ToList(),
+            serializedAdditionalRawData: null);
+    }
+
+    /// <summary> Initializes a new instance of <see cref="OpenAI.Audio.AudioTranslation"/>. </summary>
+    /// <param name="duration"> The duration of the input audio. </param>
+    /// <returns> A new <see cref="OpenAI.Audio.AudioTranslation"/> instance for mocking. </returns>
+    public static AudioTranslation AudioTranslation(string language = null, TimeSpan? duration = null, string text = null, IEnumerable<TranscribedSegment> segments = null)
+    {
+        segments ??= new List<TranscribedSegment>();
+
+        return new AudioTranslation(
+            task: default,
```

Comment:
Same here. Does this set the internal property to `InternalCreateTranslationResponseVerboseJsonTask.Translate` (a.k.a. "translate")?

Reply:
Updated both.